build.gradle

```diff
@@ -68,3 +68,19 @@ idea {
         sourceDirs += file('src/api/java')
     }
 }
+
+test {
+    workingDir = {minecraft.runDir + "/test"} // This can be set to whatever you prefer
+
+    mkdir workingDir
```

Comment:
Should not be necessary here as well.

Reply:
Yeah this is something Barometer could handle. I just need to figure out how to make Barometer into a Gradle plugin. (Same goes for https://github.com/CyclopsMC/IntegratedDynamics/pull/562#discussion_r175274508 and https://github.com/CyclopsMC/IntegratedDynamics/pull/562#discussion_r175274521)